src/components/Avatar/Avatar.tsx

```diff
@@ -116,14 +116,14 @@ export const Avatar: React.FC<AvatarProps> = (props) => {
                   ),
           }}
           style={[
-            image,
             size
               ? {
                   borderRadius: size / 2,
                   height: size,
                   width: size,
                 }
               : {},
+            image,
```

Comment:
what does the reordering of this help with and are we sure nothing else is effected? feel like there had to be a use case for why we put it in the original order

Reply:
`theme` should be able to override the default styles. If you put `image` before default styles, one won't be able to override stuff such as borderRadius, height, width through theme.